addons/forum-live-preview/addon.json

```diff
@@ -22,6 +22,16 @@
       "matches": ["editingScreens"]
     }
   ],
+  "settings": [
+    {
+      "name": "Refresh rate (ms)",
+      "id": "refresh",
+      "type": "integer",
+      "min": 250,
+      "max": 10000,
+      "default": 1000
+    }
```

Comment:
A ms setting is pretty precise. Maybe we could offer three or four options only? Just like `Auto-hiding block palette`'s "animation duration" setting

Reply:
I'll do that, but it might be a good idea for settings to support floats eventually.